<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1244</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Item1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Item2</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMpxFramePanel" name="mpxFrame" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>451</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QWPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>180</y>
     <width>371</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>safsadf</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>werqwr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>qwerqw</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sadfsaf</string>
    </property>
   </item>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>20</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPropertyTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>311</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPropertyTreeView</class>
   <extends>QTreeView</extends>
   <header>qpropertytreeview.h</header>
  </customwidget>
  <customwidget>
   <class>QMpxFramePanel</class>
   <extends>QWidget</extends>
   <header>qmpxframepanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QWPlot</class>
   <extends>QWidget</extends>
   <header>qwplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
